src/Emogrifier.php

```diff
@@ -1404,18 +1404,26 @@ private function getCssSelectorPrecedence($selector)
         $selectorKey = md5($selector);
         if (!isset($this->caches[static::CACHE_KEY_SELECTOR][$selectorKey])) {
             $precedence = 0;
-            $value = 100;
-            // ids: worth 100, classes: worth 10, elements: worth 1
-            $search = ['\\#', '\\.', ''];
-
-            foreach ($search as $s) {
+            /**
+             * @var int[] $search Keys are a regular expression part to match before a CSS name,
+             *                    values are a multiplier factor per match to weight specificity
+             */
+            $search = [
```

Comment:
This now even can be a constant instead of a local variable.

Reply:
(which would make the method more concise and move fixed data out of it)

Reply:
Have moved it to a constant.